app/core/events.py

```diff
@@ -9,6 +9,23 @@
 
 def create_start_app_handler(app: FastAPI):
     def start_app() -> None:
+        container_name, new_container_name = utils.generate_container_name("premd")
+        client = utils.get_docker_client()
+        if utils.container_exists(container_name):
```

Comment:
should there be an `else` statement following this `if`? Don't we need to create a new container if it doesn't already exist?